src/main/java/goodgenerator/blocks/tileEntity/AntimatterGenerator.java

```diff
@@ -346,19 +351,18 @@ protected MultiblockTooltipBuilder createTooltip() {
                     + "A"
                     + EnumChatFormatting.RESET
                     + EnumChatFormatting.GRAY
-                    + "*("
-                    + EnumChatFormatting.DARK_AQUA
+                    + " * ("
+                    + EnumChatFormatting.AQUA
                     + "Antimatter"
                     + EnumChatFormatting.GRAY
-                    + ")^"
+                    + ")^("
```

Comment:
Suggestion, instead of writing `A × (Antimatter)^(E)`, write `A × (Antimatter^E)`. It's better now, but it might look even better like this.


```suggestion
                    + "^"
```

Reply:
Or maybe `A × (Antimatter ^ E)`? Which looks better? (Also, which is most consistent with everything else?)